### ISSUE TITLE
Tekken tag tournament 2 visual glitches

### ISSUE THREAD

**Issue by Theoldnation** [cemu-project/Cemu#23]
Version of Cemu 2.0 (also happens in previous version)
Specs: xeon e5 2640 v3
Gpu: rx 6600
ram:16 gb ram
Api:vulkan and apparently opengl too but has a slightly different effect
Amd driver version: 22.8.1
async shaders activated
graphic packs:resolution to 1080p
multi core enabled (also happens in single core)

for unknown reasons the characters have a weird green costume or green degrade in both clothes, armos and even the model itself
(case with last phase ogre)
and also when trying to select a costume for a character, deselecting the character and selecting another the first three costumes will be the ones of the previous character
(doing some little research i had seen this bug happening to other people too)
![image](https://user-images.githubusercontent.com/76594044/186495661-8ada1c11-9e5b-4770-8f81-8e9edf2fb558.png)
![image](https://user-images.githubusercontent.com/76594044/186495768-52931d4a-fa0b-404c-9460-a216328e2d4f.png)
![image](https://user-images.githubusercontent.com/76594044/186495981-e73454be-b7ca-4e9a-b9a0-a95a5ff511c2.png)

**Comment by Theoldnation** [comment]
Game working properly in cemu 1.26.2

**Comment by ChrisHansenWithDatelineNBC** [comment]
Still an issue on 1.27.1. 

This issue does not exist on 1.26.2, but 2 player does not work properly on that version.

**Comment by Engoni** [comment]
Same issue, hope it gets fixed soon!

**Comment by leifliddy** [comment]
Please change title from ```Teken``` to ```Tekken``` as it makes this issue difficult to locate

**Comment by MagnusPrime12** [comment]
This issue is still happening with CEMU 2.0 - 6 as well. It's not fixed. I have latest ATI Radeon Drivers for a RX6700. Super annoying with a singular neon green cloth for all characters.

**Comment by Engoni** [comment]
> This issue is still happening with CEMU 2.0 - 6 as well. It's not fixed. I have latest ATI Radeon Drivers for a RX6700. Super annoying with a singular neon green cloth for all characters.

Yeah, I suggest useing the 1.26 builde, the green problem is not pressent there, but the costume select screen bug is still there, and some stages are to bright.

**Comment by MagnusPrime12** [comment]
> > This issue is still happening with CEMU 2.0 - 6 as well. It's not fixed. I have latest ATI Radeon Drivers for a RX6700. Super annoying with a singular neon green cloth for all characters.
> 
> Yeah, I suggest useing the 1.26 builde, the green problem is not pressent there, but the costume select screen bug is still there, and some stages are to bright.

Yeah, but I was hoping the developer can actual find the root cause. It's strange everything else is fixed, but this issue.

**Comment by jcchikikomori** [comment]
Same here with my Intel Xe Graphics. Using the latest 2.0 build as well

**Comment by MagnusPrime12** [comment]
They never fixed this for over a year!



Sent from my T-Mobile 5G Device



-------- Original message --------
From: John Cyrill Corsanes ***@***.***>
Date: 1/12/23 2:54 AM (GMT-05:00)
To: cemu-project/Cemu ***@***.***>
Cc: MagnusPrime12 ***@***.***>, Comment ***@***.***>
Subject: Re: [cemu-project/Cemu] [Tekken Tag Tournament 2] Visual glitches after Cemu 1.26.2 (Issue #23)


Same here with my Intel Xe Graphics. Using the latest 2.0 build as well

**Comment by leifliddy** [comment]
I just compiled Cemu from source (for Fedora) using this container-based method
https://github.com/leifliddy/podman-cemu-builder/tree/main

And for the first time in over a year -- the visual glitches are finally gone. 
The characters are no longer highlighted in fluorescent green-colored clothing. The last time I complied Cemu was on Aug 5.
So something clearly happened within the last month or so to resolve this issue. 

Hopefully, you guys are seeing the same thing and we can close this issue.

**Comment by MagnusPrime12** [comment]
YES!! I verified that it's fixed. You can close this. After a year, finally fixed!! Thanks to the developers.

**Comment by Engoni** [comment]
> YES!! I verified that it's fixed. You can close this. After a year, finally fixed!! Thanks to the developers.

I dont get green characters any more (hurray) but some stages are still wery overly bright and saturated (auto HDR fixes it a bit, but not completely) and I still have the character select problem!

**Comment by ghostporkchop** [comment]
The costumes/models in Tekken Tag 2 Tournament are glitching after entering a mode then leaving to go play another mode. This is for cemu 2.0 and above I hope there can be a fix for this game.
